sats-dna/src/main/kotlin/com/sats/dna/components/CompletedWorkoutListItem.kt

```diff
@@ -39,42 +40,69 @@ fun CompletedWorkoutListItem(
     timestamp: String,
     title: String,
     location: String?,
-    numberOfCommentsLabel: String,
+    numberOfComments: String,
```

Comment:
Should we change this into a `String`, too, now? Since we don't show the “comments” string anymore.

Reply:
YOLO.